Gemfile

```diff
@@ -87,7 +87,7 @@ end
 # previously-passing codebase start failing. Thus, we need to lock to a specific
 # version and bump it from time to time.
 # Also, there's no support for windows
-if RUBY_VERSION >= '2.4.0' && !Gem.win_platform?
+if RUBY_VERSION >= '2.4.0' && !Gem.win_platform? && RUBY_PLATFORM != 'aarch64-linux'
```

Comment:
I think it makes sense to rather have this be `RUBY_PLATFORM =~ /^x86_64-(darwin|linux)/` instead, WDYT?

Angle is making it work out of the box for other platforms such as freebsd or ppc64le, and consistency with what `sorbet` is available on.

Reply:
Fixed in 249a803dd7b849b8bc94167add5fb5a1a4c875ae